doc/UsersGuide/CustomizingTheWorkflow/InputOutputFiles.rst

```diff
@@ -226,14 +226,14 @@ A set of input files, including static (fix) data and raw initial and lateral bo
 Static Files
 --------------
 
-Static files are available in the `"fix" directory <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/fix/>`__ of the SRW App Data Bucket. Users can download the full set of fix files as a tar file:
+Static files are available in the `"fix" directory <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#experiment-user-cases/release-public-v3.0.0/fix/>`__ of the SRW App Data Bucket. Users can download the full set of fix files as a tar file:
```

Comment:
@jdkublnick Should this link to either `develop` (https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20250321/fix/) or to the tar file (https://noaa-ufs-srw-pds.s3.amazonaws.com/experiment-user-cases/release-public-v3.0.0/out-of-the-box/fix_data.tgz)?